### PR TITLE
misc documentation updates

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: docs/user_manual/barcode_label_printers.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Barcode Label Printers
 
 MISO pages can be printed to any printer that is installed on the computer you are using. No extra configuration is
-required to do this. Printing barcode labels to label printers does require some configuration, however.
-
+required to do this. Printing barcode labels to label printers does require some configuration, however. Brady and
+Zebra label printers are supported.
```
Comment (avarsava) [inline docs/user_manual/barcode_label_printers.md:5]: Specific model #s?

Comment (djcooke) [reply docs/user_manual/barcode_label_printers.md:5]: To my knowledge, all should work

Comment (avarsava) [reply docs/user_manual/barcode_label_printers.md:5]: In that case I think it still might be more accurate list the models we've verified to work and write that the rest should work but have not been tested, but if you do not have that information this is fine

Comment (djcooke) [reply docs/user_manual/barcode_label_printers.md]: Yeah, good idea. Added

Comment (avarsava) [reply docs/user_manual/barcode_label_printers.md]: :+1:

**Pull request merged**